mantidimaging/core/utility/version_check.py

```diff
@@ -5,92 +5,153 @@
+
+    def __init__(self):
+        self._version = __version__
+        self._conda_installed_version = None
+        self._conda_installed_label = None
+        self._conda_available_version = None
+
+        # To test update warning message uncomment
+        # self._use_test_values(False)
+
+    def _retrieve_versions(self) -> None:
+        self._retrieve_conda_installed_version()
+        self._retrieve_conda_available_version()
+
+    def _use_test_values(self, uptodate: bool = True) -> None:
+        """Avoid fetching version info"""
+        self._conda_installed_version = "1.0.0_1"
+        self._conda_installed_label = "main"
+        if uptodate:
+            self._conda_available_version = "1.0.0_1"
+        else:
+            self._conda_available_version = "2.0.0_1"
+
+    def show_versions(self) -> None:
+        print(f"Mantid imaging {self.get_version()}")
+        print(f"conda_installed_version {self.get_conda_installed_version()}")
+        print(f"conda_installed_label {self.get_conda_installed_label()}")
+        print(f"conda_available_version {self.get_conda_available_version()}")
+
+    def get_version(self) -> str:
+        """Get built in version"""
+        return self._version
+
+    def get_conda_installed_version(self) -> Optional[str]:
+        """Get version number of installed package from conda"""
+        if self._conda_installed_version is None:
+            self._retrieve_conda_installed_version()
+        return self._conda_installed_version
+
+    def get_conda_installed_label(self) -> Optional[str]:
+        """Get with 'main' or 'unstable' from conda installed package"""
+        if self._conda_installed_label is None:
+            self._retrieve_conda_installed_version()
+        return self._conda_installed_label
+
+    def get_conda_available_version(self) -> Optional[str]:
+        """Get latest version number from conda"""
+        if self._conda_available_version is None:
+            self._retrieve_conda_available_version()
+        return self._conda_available_version
+
+    def is_conda_uptodate(self) -> bool:
+        """Check if up to date with lasted version in conda"""
+        if (self.get_conda_installed_version() == "" or self.get_conda_available_version() == ""):
+            # If unable to get conda versions then assume everything is good
+            return True
+
+        return _version_is_uptodate(_parse_version(self.get_conda_installed_version()),
+                                    _parse_version(self.get_conda_available_version()))
+
+    def conda_update_message(self) -> tuple:
+        suffix = self.get_conda_installed_label()
+
+        msg = f"Not running the latest Mantid Imaging{suffix}.\n"
+        msg += f"Found version {self.get_conda_installed_version()}, "
+        msg += f"latest: {self.get_conda_available_version()}.\nPlease check the terminal for an update command!"
+
+        detailed = f"Not running the latest Mantid Imaging{suffix}.\n"
```

Comment:
```suggestion
        detailed = f"Not running the latest Mantid Imaging-{suffix}.\n"
```
I think this needs some separator before the suffix as it looks like this without one:

![image](https://user-images.githubusercontent.com/9135965/101480358-7c8edd00-394b-11eb-93db-d53c8c40f325.png)

It may be worth not appending "main" at all, but only add "unstable" for the unstable version